pkg/engine/kube.go

```diff
@@ -23,81 +24,158 @@ type YamlMeta struct {
 	Kind       string `yaml:"kind"`
 }
 
-func kubePodman(ctx context.Context, path string) error {
-	klog.Infof("Creating podman container from %s using kube method", path)
-
-	kubeYaml, err := ioutil.ReadFile(path)
+func kubePodman(ctx context.Context, path *string, prev *string) error {
+	if path != nil {
+		klog.Infof("Creating podman container from %s using kube method", *path)
+	}
+	ctx, err := bindings.NewConnection(ctx, "unix://run/podman/podman.sock")
```

Comment:
ctx should be conn here, the ctx is passed in from EngineMethod

Reply:
oops nm, I'm wrong :) ctx is returned from NewConnection, so should be 
```
var err error
ctx, err, = bindings.NewConnection(ctx, ...)
```

Reply:
:shrug: I keep seeing this error: `Client not set in context`, have you seen it?  we'll look at it next week